infrastructure/code/template/details.html

```diff
@@ -438,91 +452,140 @@ <h2 class="severity-title">{{.IssueTitle}}</h2>
   </header>
 
   <main class="code-issue-container">
-    <!-- issue overview -->
-    <article class="issue-overview">
-      <div class="overview-text">
-        {{.IssueOverview}}
+    <!-- Tabs Issue Overview Container -->
+    <article class="code-issue-panel">
+      <div class="main-tabs-nav">
+        {{if .IsIgnored}}
+        <span data-tab="ignore-details" id="ignore-details-tab" class="tab-item is-selected ignore-details-tab">
+          Ignore Details
+        </span>
+        <span data-tab="fix-analysis" id="fix-analysis-tab" class="tab-item fix-analysis-tab">
+          Fix Analysis
+        </span>
+        <span data-tab="vuln-overview" id="vuln-overview-tab" class="tab-item vuln-overview-tab">
+          Vulnerability Overview
+        </span>
+        {{else}}
+        <span data-tab="fix-analysis" id="fix-analysis-tab"
+          class="tab-item is-selected sn-fix-analysis fix-analysis-tab">
+          Fix Analysis
+        </span>
+        <span data-tab="vuln-overview" id="vuln-overview-tab" class="tab-item sn-vuln-overview vuln-overview-tab">
+          Vulnerability Overview
+        </span>
+        {{end}}
       </div>
-    </article>
```

Comment:
Some class names were brought over from the current mark-up in the `vscode-plugin` project, and it's possible that not all of them are needed.